core/DataAccess/Model.php

```diff
@@ -118,13 +129,21 @@ public function deleteArchivesWithPeriod($numericTable, $blobTable, $period, $da
         $query = "DELETE FROM %s WHERE period = ? AND ts_archived < ?";
         $bind  = array($period, $date);
 
-        Db::query(sprintf($query, $numericTable), $bind);
+        $queryObj = Db::query(sprintf($query, $numericTable), $bind);
+        $deletedRows = $queryObj->rowCount();
 
         try {
-            Db::query(sprintf($query, $blobTable), $bind);
+            $queryObj = Db::query(sprintf($query, $blobTable), $bind);
+            $deletedRows += $queryObj->rowCount();
         } catch (Exception $e) {
             // Individual blob tables could be missing
+            $this->logger->debug("Unable to delete archives by period from {blobTable}.", array(
+                'blobTable' => $blobTable,
+                'exception' => $e
+            ));
         }
+
+        return $deletedRows;
```

Comment:
> I think tests for just the Model would be somewhat redundant considering all it does is execute SQL.

It sounds like there could be no bugs, and like nobody would ever want to change that code :)